### DISCUSSION TITLE
Who's hiring <Remote Only>? (Beta)

### DISCUSSION THREAD

**Discussion by zhyd1997** [SoftMaple/Editor#62] [Ideas]
### What is this?
**It's inspired by [Next.js disccusions](https://github.com/vercel/next.js/discussions/)**.

This discussion is for companies looking for **software engineers** that can work remotely from **China**.
Both _**permanent**_ positions and _**contract**_ work are welcome.

### Companies
Please use the following format followed by a brief description of the role:

> COMPANY_NAME | ROLE | Full-time | ¥20-40k base | LINK
>
> Write some details here:

Please **only** post if you personally are part of the hiring company—no recruiting firms or job boards. Only one post per company.

Good luck with recruiting!

**Comment by yangwao** [comment]
KodaDot | Senior Frontend | Part/full-time | $40-100k base | https://kodadot.xyz/jobs

We are open source NFT marketplace and we working on Kusama network, feel free to ask more at our Discord. Our repository https://github.com/kodadot/nft-gallery

**Reply by zhyd1997** [reply]
Thanks, I will promote it!
